app/views/shared/_open_on_assistant_modal.html.erb

```diff
@@ -1,12 +1,17 @@
-<iframe id="js-hidden-iframe" src="about:blank" style="display:none"></iframe>
+<% unless disabled %>
+  <iframe id="js-hidden-iframe" src="about:blank" style="display:none"></iframe>
+<% end %>
 <div class="mt-3 pr-2">
   <details class="dropdown details-reset details-overlay d-inline-block">
-    <summary id="js-download-repos-btn" class="btn" aria-haspopup="true">
+    <summary id=<%=  disabled ? "" : "js-download-repos-btn" %> class="btn <%= disabled ? "disabled" : "" %>" aria-haspopup="true">
```

Comment:
Curious if you think there should be an aria-label and a tooltip when this is disabled to mention why it's disabled?

Reply:
That's a great idea, feel free to branch off my branch in #1665